### PR TITLE
As per last review

### PR DESCRIPTION
**Pull request by ssat335**
Deleted  redundant cmgui file.
Added synopsis, made sure all the contents are properly structured in README.rst
Deleted lines from .py that were obsolete.

### CODE REVIEW

File: src/Python/monodomain_2D_HH.py [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+	cellml_file = sys.argv[1]
+else:
+	cellml_file = 'HodgkinHuxley1952.cellml'
+
+#DOC-START create cellml environment
+# Create the CellML environment
+cellML = iron.CellML()
+cellML.CreateStart(cellMLUserNumber, region)
+# Import a Nobel 98 cell model from a file
+noble98Model = cellML.ModelImport(cellml_file)
```
Comment (hsorby) [inline src/Python/monodomain_2D_HH.py]: This variable name isn't very helpful perhaps `cellmlModel` would be better!

**Review by hsorby — CHANGES_REQUESTED** [review]
Looking really good now, just one more minor change and we can merge this in.

File: src/Python/monodomain_2D_HH.py [hunk 2]
```diff
@@ -161,16 +161,16 @@
...
-	cellml_file = 'HodgkinHuxley1952.cellml'
+	cellmlModel = 'HodgkinHuxley1952.cellml'
 
 #DOC-START create cellml environment
 # Create the CellML environment
 cellML = iron.CellML()
 cellML.CreateStart(cellMLUserNumber, region)
 # Import a Nobel 98 cell model from a file
-noble98Model = cellML.ModelImport(cellml_file)
+noble98Model = cellML.ModelImport(cellmlModel)
```
Comment (hsorby) [inline src/Python/monodomain_2D_HH.py]: Not that variable! I was meaning change `noble98Model` to `cellmlModel` the `cellml_file` name is fine.

**Review by hsorby — CHANGES_REQUESTED** [review]

File: src/Python/monodomain_2D_HH.py (hunk 2 above)
Comment (chrispbradley) [reply src/Python/monodomain_2D_HH.py]: What is wrong with noble98Model? Seems like the variable name is exactly what it is?

Comment (hsorby) [reply src/Python/monodomain_2D_HH.py]: Except that it's a hodgkin huxley model.

Comment (hsorby) [reply src/Python/monodomain_2D_HH.py]: It could be any model if passed in through the command line.

Comment (chrispbradley) [reply src/Python/monodomain_2D_HH.py]: Ahhhh, point.

Comment (ssat335) [reply src/Python/monodomain_2D_HH.py]: My bad!! Done.

**Review by hsorby — APPROVED** [review]

**Pull request merged**